testing/mysql/integration_test.go

```diff
@@ -118,7 +118,7 @@ func TestIntegration_MYSQLDUMP_SimpleUse(t *testing.T) {
 	if err != nil {
```

Comment:
We should add a simple test for skip-foreign-keys -- maybe a follow-up PR?

Reply:
Agreed, I'll put it up in the next PR.